packages/frontend/src/utils/account.js

```diff
@@ -6,4 +7,17 @@ export const validateEmail = (email) => {
 };
 
 export const isImplicitAccount = (accountId) =>
-    accountId && accountId.length === 64 && !accountId.includes('.');
+    accountId && accountId.length === IMPLICINT_ACCOUNT_MAX_LENGTH && !accountId.includes('.');
+
+export const shortenAccountId = (id, startChars = 8, endChars = 8) => {
+    const separator = '...';
```

Comment:
Should be move out of scope as `const SEPARATOR = '...';`

Reply:
I thought about this, so we won't need to recreate this var. But at the end didn't move it, because it seems out of context). I consider it's not important and we can save it right above this func.